packages/@aws-cdk-testing/cli-integ/lib/package-sources/repo-source.ts

```diff
@@ -75,13 +75,14 @@ const YARN_MONOREPO_CACHE: Record<string, any> = {};
   *
   * Cached in YARN_MONOREPO_CACHE.
   */
-async function findYarnPackages(root: string): Promise<Record<string, string>> {
+export async function findYarnPackages(root: string): Promise<Record<string, string>> {
```

Comment:
This method is meant to find the local CDK packages and use them when running the tests. It was broken, and the result was that it always retrieved the latest CDK release, instead of using your local code. This fixes that.